adapters/adnuntius/adnuntius.go

```diff
@@ -265,6 +108,13 @@ func (a *adapter) generateRequests(ortbRequest openrtb2.BidRequest) ([]*adapters
 		return nil, []error{fmt.Errorf("failed to parse site Ext: %v", err)}
 	}
 
+	var extUser openrtb_ext.ExtUser
+	if ortbRequest.User != nil && ortbRequest.User.Ext != nil {
+		if err := jsonutil.Unmarshal(ortbRequest.User.Ext, &extUser); err != nil {
+			return nil, []error{fmt.Errorf("failed to parse Ext User: %v", err)}
```

Comment:
Please add a supplemental JSON test case where user.ext is invalid(if not already added).

Reply:
@pm-isha-bharti thank you for your comments! 
We don't ask to cover unmarshal errors. JSON test should have a valid format to be loaded to the test suit and if JSON itself is invalid it will never reach this point. 
We need these error handling for real requests, because `user.ext` may be untouched and unverified until it reaches the bidder code.
Hope it makes sense :)